src/utils/fhir.js

```diff
@@ -152,7 +157,10 @@ function refreshAllKnowledgeArtifacts() {
  * @param {*} server - the server to refresh artifacts from
  */
 function refreshKnowledgeArtifact(server) {
-  getResources(server.endpoint, 'PlanDefinition');
+  getResources(server.endpoint, 'PlanDefinition').then(bundle =>
+    // Create/Update subscriptions from PlanDefinitions
+    subscriptionsFromBundle(bundle, server.endpoint, 'admin')
```

Comment:
The original intent of this function was just to create the subscription resources. Since it now does more can you update the description of the function please